environment.yaml

```diff
@@ -0,0 +1,16 @@
+name: earth-env-data-science-book
+channels:
+    - conda-forge
+dependencies:
+  - xarray
+  - xmip
+  - cf_xarray # remove once https://github.com/jbusecke/xMIP/issues/274 is fixed
+  - gcsfs
+  - intake-esm
+  - matplotlib
+  - nc-time-axis
+  - numpy
+  - jupyterlab
+  - pip
+  - pip:
+    - -r requirements.txt
```

Comment:
Where is this environment.yaml being used? Are you sure it is comprehensive for the entire book?

An alternative would be to point to the pangeo docker image.

Reply:
I am fairly sure it is not comprehensive. I just threw it together for myself (and then did not think about committing it). Ill switch to the docker image.